src/include/stir/IO/InputStreamFromROOTFile.inl

```diff
@@ -135,11 +135,13 @@ InputStreamFromROOTFile::set_exclude_random_events(bool val)
     exclude_randoms = val;
 }
 
+#if STIR_ROOT_ROTATION_AS_V4
 void
 InputStreamFromROOTFile::set_detectors_offset(int val)
 {
     offset_dets = val;
 }
+#endif
```

Comment:
Nesting the method inside of `#if STIR_ROOT_ROTATION_AS_V4` is okay?

Reply:
sure